Templates/make_jenkins_EC2-Agent-Windows-instance.tmplt.json

```diff
@@ -0,0 +1,1613 @@
+                                                        "Fn::Join": [
+                                                            "",
+                                                            [
+                                                                " --config \\\"",
+                                                                {
+                                                                    "Ref": "WatchmakerConfig"
+                                                                },
+                                                                "\\\""
+                                                            ]
+                                                        ]
+                                                    },
+                                                    ""
+                                                ]
+                                            },
+                                            {
+                                                "Fn::If": [
+                                                    "UseEnvironment",
+                                                    {
+                                                        "Fn::Join": [
+                                                            "",
+                                                            [
+                                                                " --env \\\"",
+                                                                {
+                                                                    "Ref": "WatchmakerEnvironment"
+                                                                },
+                                                                "\\\""
+                                                            ]
+                                                        ]
+                                                    },
+                                                    ""
+                                                ]
+                                            },
+                                            {
+                                                "Fn::If": [
+                                                    "UseOuPath",
+                                                    {
+                                                        "Fn::Join": [
+                                                            "",
+                                                            [
+                                                                " --ou-path \\\"",
+                                                                {
+                                                                    "Ref": "WatchmakerOuPath"
+                                                                },
+                                                                "\\\""
+                                                            ]
+                                
```

Comment:
Note: due to its ever-growing nature, it's best to have `${JENKINS_HOME}` on its own EBS (separate from the "root" EBS volume &mdash; what's mounted as `C:`). The standard CFn logic will mount (likely) the secondary EBS at `D:`. You likely want to add logic to your `appscript` and `chainscript` to account for this.

Obviously, you'll want to validate that everything works when you place `${JENKINS_HOME}` on `D:` (or, if you choose to update your scripts to rename the secondary EBS from `D:` to, say, `J:`, then _that_ location).